sdk/tests/deqp/functional/gles3/es3fLifetimeTests.js

```diff
@@ -100,8 +100,12 @@ es3fLifetimeTests.ScaleProgram.prototype.draw = function(vao, scale, tf, dst) {
 es3fLifetimeTests.ScaleProgram.prototype.setPos = function(buffer, vao) {
     gl.bindBuffer(gl.ARRAY_BUFFER, buffer);
     gl.bindVertexArray(vao);
-    if (buffer)
+    if (buffer) {
         gl.vertexAttribPointer(this.m_posLoc, NUM_COMPONENTS, gl.FLOAT, false, 0, 0);
```

Comment:
Right. Instead it has an offset that points to the bound buffer.

Still, why do we need to delete the buffer?  The C++ code doesn't.  Does that cause a behavior change? I try to understand the motivation.